### DISCUSSION TITLE
Logo displayed in explorer but not Phantom.

### DISCUSSION THREAD

**Discussion by MikeVEMP** [phantom/docs#180] [Help & Support]
### Summary

I deployed a token earlier today and uploaded the metadata. The correct details are applied in explorers like Solscan and the the native Solana explorer. However in Phantom it is still displayed as "Unknown Token".

On devnet the same process updated very quickly, and my hope is that mainnet just takes longer to update in-app, rather than being an issue with my metadata itself.

For reference the token address and metadata can be found here: https://solscan.io/token/ANTT2Ve8c3JC6fAksFxUGtByzEHmJs49EupP7htA5W7j#metadata

### Steps to Reproduce

Token information can be viewed here: https://solscan.io/token/ANTT2Ve8c3JC6fAksFxUGtByzEHmJs49EupP7htA5W7j#metadata

Screenshot of Phantom Wallet attached.
<img width="345" alt="Screenshot 2024-02-21 at 15 07 43" src="https://github.com/phantom/docs/assets/126326457/b2751d06-4277-4188-a6b3-c335ba121e32">

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@MikeVEMP Hello,

This token now appears to be showing in Phantom as expected.
